localMatchApp/headers/FunctionsEvent.h

```diff
@@ -0,0 +1,45 @@
+#ifndef FUNCTIONS_H
+#define FUNCTIONS_H
+
+#include "ClassEvent.h"
+
+/*!
+ * The class of functions is created to
+ * be able to develop the functions
+ * that will manage the database
+ */
+class FunctionsEvent
```

Comment:
This name does not reflect the utility of the class. Please, think on a new name for this class.